tests/integration/validations/factory-general-test.js

```diff
@@ -1040,7 +1040,7 @@ module('Integration | Validations | Factory - General', function(hooks) {
     assert.equal(child.get('validations.errors.length'), 1);
   });
 
-  test('call super in validations class with no super property', function(assert) {
+  skip('call super in validations class with no super property', function(assert) {
```

Comment:
this is also failing in Ember 3.11: https://github.com/offirgolan/ember-cp-validations/pull/656